cmd/tailscale/cli/serve_test.go

```diff
@@ -49,6 +49,10 @@ func TestCleanMountPoint(t *testing.T) {
 	}
 }
 
+// strp returns a pointer to the string s.
+// Used for tests that need a pointer to a string.
+func strp(s string) *string { return &s }
```

Comment:
We have ptrTo defined in a few places. I think I added it in `set_test.go` so you should be able to use it instead of this

Reply:
Moved it to `cli/cli.go`